Practical Example/RWTakeHome/RWTakeHome/Extenstion/ViewController+Ext.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ViewController+Ext.swift
+//  RWTakeHome
+//
+//  Created by Nick Nguyen on 10/7/20.
+//
+
+import UIKit
+
+extension TutorialViewController {
+  
+  func reloadDataOnMainThread() {
+    DispatchQueue.main.async {
+      self.collectionView.reloadData()
+    }
+  }
+
+   func getArticles() {
+    networkService.fetchTutorials(ofType: .article) {  (result) in
+      guard let result = try? result.get() else { fatalError()}
```

Comment:
I assume you only did this because it's a test project, but an interviewer would still like to see how you handle errors. `fatalError` will crash the app if you get an error. What if the user doesn't have internet connection, or the server is down?

Reply:
You are right. I need to handle errors. I was rush. Thank you!